README.md

```diff
@@ -8,4 +8,81 @@ This repo holds packaging automation that builds debs from `kanidm/kanidm` and f
 - Changes in the public signing key need to be addressed in both `kanidm/kanidm_ppa` & this repo.
 
 For instructions how to use this repo for manual builds, see the book at:
-https://kanidm.github.io/kanidm/stable/packaging/debian_ubuntu_packaging.html
+<https://kanidm.github.io/kanidm/stable/packaging/debian_ubuntu_packaging.html>
+
+## Release process
+
+To cut a new release after upstream releases perform the following steps:
```

Comment:
Mainly for the comma, because I'm that kinda grammar-kid. 😄 

```suggestion
To cut a new release after upstream does, perform the following steps:
```